dot/network/notifications.go

```diff
@@ -158,6 +158,8 @@ func (s *Service) createNotificationsMessageHandler(
 
 			hs, ok := msg.(Handshake)
 			if !ok {
+				// this error will never happen.
+				// Handshake interface and NotificationMessage interfaces are same.
```

Comment:
Should we then remove the `, ok` in `hs, ok := msg.(Handshake)` so it panics in case it ever happens?

Reply:
Arhhmm, I am not really sure about that. I would prefer not to remove this.

I feel like NotificationMessage and Handshake happens to have the same interface at the moment, but they don't have to.  Also, not a big fan of panicking.

Reply:
Then let's remove that comment.

Panicing is useful for programming errors, so it doesn't go unnoticed in some error log line.
I also was team anti-panic, but it really has its use for programming errors (not from user/IO of course)